li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/largemessage/LargeMessageSegment.java

```diff
@@ -33,7 +34,7 @@
   public final ByteBuffer payload;
   // The segment information over head bytes when serialize.
   public static final int SEGMENT_INFO_OVERHEAD = 16 + Integer.BYTES + Integer.BYTES + Integer.BYTES;
-  public static final byte CURRENT_VERSION = 0;
+  public static final byte CURRENT_VERSION = LargeMessageHeaderValue.V3;
```

Comment:
The current version should still be the same as before because your patch does not bump major version yet. So, if you update current_version to V3 in this patch (lkc_currentVersion+1), then existing consumers (<= lkc_currentVersion) will not be able to parse the new header version.

Reply:
Thanks for the comment. I agree with you about the version should keep same for current consumers. I will fix this.

Reply:
It will also be useful to add a `TODO` comment here explaining when this value should be change to `V3`

Reply:
`TODO` added.